exotica_core/include/exotica_core/kinematic_element.h

```diff
@@ -121,6 +121,8 @@ class KinematicElement
     KDL::Frame generated_offset = KDL::Frame::Identity();
     bool is_trajectory_generated = false;
     std::vector<double> joint_limits;
+    std::vector<double> velocity_limit;
+    std::vector<double> acceleration_limit;
```

Comment:
Change to plural ```velocity_limits``` and ```acceleration_limits``` to match the ```joint_limits```

Reply:
I gave a different input earlier, and here is the rationale: For a KinematicElement, the position limits are plural: lower, upper. Acceleration and velocity limit are singular - and should be only a double and not a vector: It's a single value.